doc/coding-conventions.xml

```diff
@@ -191,6 +191,18 @@ args.stdenv.mkDerivation (args // {
  <section xml:id="sec-package-naming">
   <title>Package naming</title>
 
+  <para>
+    The key words
+    <emphasis>must</emphasis>, <emphasis>must not</emphasis>,
+    <emphasis>required</emphasis>,
+    <emphasis>shall</emphasis>, <emphasis>shall not</emphasis>,
+    <emphasis>should</emphasis>, <emphasis>should not</emphasis>,
+    <emphasis>recommended</emphasis>, <emphasis>may</emphasis>,
+    and <emphasis>optional</emphasis> in this section
+    are to be interpreted as described in
+    <link xlink:href="https://tools.ietf.org/rfc/rfc2119.txt">RFC 2119</link>.
```

Comment:
Tiny nitpick suggestions:

```suggestion
    <link xlink:href="https://tools.ietf.org/html/rfc2119">RFC 2119</link>.
    Only <emphasis>emphasized</emphasis> words are to be interpreted in this way.
```

The new sentence is from [RFC 8174](https://tools.ietf.org/html/rfc8174), as it's typical to sneak some of these words into text without meaning to have that strict interpretation.  Using html links to IETF RFCs makes it a little easier to e.g. notice that it's been updated upstream (like in this case).

Reply:
OK updated.